packages/runtime/src/useCases/transport/messages/SendMessage.ts

```diff
@@ -58,23 +58,30 @@ export class SendMessageUseCase extends UseCase<SendMessageRequest, MessageDTO>
 
     private async validateMessageContent(content: any, recipients: string[]) {
         const transformedContent = Serializable.fromUnknown(content);
-        if (!(transformedContent instanceof Request)) return;
+        if (transformedContent instanceof Request) {
+            if (!transformedContent.id) return RuntimeErrors.general.invalidPropertyValue("The Request must have an id.");
 
-        if (!transformedContent.id) return RuntimeErrors.general.invalidPropertyValue("The Request must have an id.");
+            const localRequest = await this.outgoingRequestsController.getOutgoingRequest(transformedContent.id);
+            if (!localRequest) return RuntimeErrors.general.recordNotFound(Request);
 
-        const localRequest = await this.outgoingRequestsController.getOutgoingRequest(transformedContent.id);
-        if (!localRequest) return RuntimeErrors.general.recordNotFound(Request);
+            if (!_.isEqual(transformedContent.toJSON(), localRequest.content.toJSON())) {
+                return RuntimeErrors.general.invalidPropertyValue("The sent Request must have the same content as the LocalRequest.");
+            }
 
-        if (!_.isEqual(transformedContent.toJSON(), localRequest.content.toJSON())) {
-            return RuntimeErrors.general.invalidPropertyValue("The sent Request must have the same content as the LocalRequest.");
+            if (recipients.length > 1) return RuntimeErrors.general.invalidPropertyValue("Only one recipient is allowed for sending Requests.");
+
+            const recipient = CoreAddress.from(recipients[0]);
+            if (!recipient.equals(localRequest.peer)) return RuntimeErrors.general.invalidPropertyValue("The recipient does not match the Request's peer.");
+            return;
+        } else if (
+            transformedContent instanceof Mail ||
+            transformedContent instanceof ResponseWrapper ||
+            transformedContent instanceof Notification ||
+            transformedContent instanceof ArbitraryMessageContent
+        ) {
+            return;
```

Comment:
I think an early return is preferred here, i.e. test for the valid types in the beginning and throw an error in case of an invalid type. Then, continue with the validation for Requests.

Reply:
putting the happy path first allows for cleaner ifs

Reply:
@Magnus-Kuhn I am 100% with Milena. we don't use `else if` in the code if avoidable and this is the perfect place to avoid it.